specification/edgeorderpartner/resource-manager/readme.md

```diff
@@ -0,0 +1,41 @@
+# EdgeOrderPartner
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for EdgeOrderPartner.
+
+
+
+---
+## Getting Started
+To build the SDK for EdgeOrderPartner, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the EdgeOrderPartner API.
+
+``` yaml
+title: Edge Order Partner API's
```

Comment:
It is recommended to take a SDK friendly `title`. It is usually used as entry point of the SDK. It usually cannot display space or quote.

E.g. compute https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md#basic-information
results in SDK class https://docs.microsoft.com/en-us/python/api/azure-mgmt-compute/azure.mgmt.compute.computemanagementclient?view=azure-python

You can put the detail in `description`.